src/NextGen/Framework/FormTemplates/Registrars/FormTemplateRegistrar.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Give\NextGen\Framework\FormTemplates\Registrars;
+
+use Give\Framework\Exceptions\Primitives\InvalidArgumentException;
+use Give\Framework\PaymentGateways\Exceptions\OverflowException;
```

Comment:
Should we be re-using Payment Gateway exceptions here?

Reply:
nope lol updated with new one (but we should probably add OverflowException to our give exceptions)

https://github.com/impress-org/givewp-next-gen/compare/a5e96c062bf7...73b20128278e